templates/common/packages/tsconfig/nextjs.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "tsconfig/nextjs.json",
+  "extends": "./base.json",
```

Comment:
@achaljain @vinodloha "templates/common/packages/tsconfig/nextjs.json" this file does not need any change as we have removed "templates/common/packages/tsconfig" folder itself. I test other changes done by Achal and it's working fine.